addons/skin.estuary/xml/DialogPVRChannelGuide.xml

```diff
@@ -75,18 +75,27 @@
 							<visible>!String.IsEmpty(ListItem.Plot) + Control.HasFocus(11)</visible>
 							<animation effect="fade" start="0" end="100" time="200">Focus</animation>
 							<control type="image">
-								<left>782</left>
+								<left>772</left>
 								<top>-100</top>
-								<width>900</width>
+								<width>1210</width>
 								<height>350</height>
 								<texture border="40">buttons/dialogbutton-nofo.png</texture>
 							</control>
+							<control type="image">
+								<left>805</left>
+								<top>-65</top>
+								<width>320</width>
+								<height>280</height>
+								<aspectratio align="center" aligny="center">keep</aspectratio>
+								<texture fallback="$INFO[Listitem.Icon]">$INFO[Listitem.EpgEventIcon]</texture>
+							</control>                            
 							<control type="textbox">
-								<left>815</left>
+								<left>1135</left>
 								<top>-70</top>
 								<width>760</width>
 								<height>290</height>
 								<label>$INFO[ListItem.Plot]</label>
+                                <autoscroll delay="2000" time="2000" repeat="10000">Skin.HasSetting(AutoScroll)</autoscroll>
```

Comment:
No way to stroll in unrelated changes. Keep the changes to the topic of this PR, pleaes.

Reply:
ok